db/migrate/20231005162336_add_address_to_veteran_representative.rb

```diff
@@ -0,0 +1,23 @@
+class AddAddressToVeteranRepresentative < ActiveRecord::Migration[6.1]
+  def change
+    add_column :veteran_representatives, :address_line_1, :string
```

Comment:
Having all of these columns let's us adhere to [Lighthouse guidelines](https://department-of-veterans-affairs.github.io/lighthouse-api-standards/naming-and-formatting/addresses/), match existing address validation patterns in `lib/va_profile/models/validation_address.rb`, and lets us meet our search needs.